python/common/python/math.java

```diff
@@ -0,0 +1,35 @@
+package python;
+
+@org.python.Module(
+        __doc__ =
+                "math"
+)
+public class math extends org.python.types.Module {
+    public math() {
+        super();
+    }
+
+    @org.python.Attribute
+    public static org.python.Object __file__ = new org.python.types.Str("python/common/python/math.java");
+    @org.python.Attribute
+    public static org.python.Object __loader__ = org.python.types.NoneType.NONE;  // TODO
+    @org.python.Attribute
+    public static org.python.Object __name__ = new org.python.types.Str("math");
+    @org.python.Attribute
+    public static org.python.Object __package__ = new org.python.types.Str("");
+    @org.python.Attribute
+    public static org.python.Object __spec__ = org.python.types.NoneType.NONE;  // TODO
+
+    @org.python.Method(
+            __doc__ = "",
```

Comment:
It's a good idea to add _at least_ the name of the function here in the doc tag, even better something like ``sqrt(number) -> calculates the square root of the number `number` ``